File: Java/linearsearch.java
import java.util.*;
public class linearsearch {
    public static void main(String args[]){
        Scanner sc= new Scanner(System.in);
        int n=sc.nextInt();
        int x[] = new int [n] ;
        int pos = 0;
        System.out.println("Enter the values in the arrray: ");
        for(int i=0; i<n; i++){
            x[i] = sc.nextInt();
        }

        System.out.println("Enter the number you want to search: ");
        int num = sc.nextInt();
    
        for(int i=0; i<n ; i++){
            if(x[i] == num) {
                pos=i+1;
            }
        }
        System.out.println("The position of the entered number is "+pos);
        sc.close();
    }
}

File: Data Structure/linearsearch.cpp
#include<iostream>
using namespace std;

int main(){
    int n,num,arr[n+1],flag,pos;
    cout<<"Enter the number of the elements: "<<endl;
    cin>>n;
    cout<<"Enter the element of the array: "<<endl;
    for(int i=0;i<n;i++){
        cin>>arr[i];
    }

    cout<<"Enter the number you want to search: "<<endl;
    cin>>num;
    for(int i=0;i<n;i++){
        if(num==arr[i]){
            flag=1;pos=i;
            cout<<"The number is at position: "<<pos<<endl;
        }
    }
    if(flag!=1){
        cout<<"The number is not present in the array:";
    }
}